pkg/config/application_defaults.go

```diff
@@ -16,7 +16,8 @@ const (
 	ApplicationsDefaultsConfigFileName = "defaults.yml"
 )
 
-// ApplicationConfig contains applications to install during boot
+// ApplicationDefaultsConfig contains optional defaults for applications installed via helmfile / helm 3 which are
+// typically discovered from the Version Stream
```

Comment:
Same as above squash this commit into here [`fix: add version stream defaults for apps in helmfile`](https://github.com/jenkins-x/jx/pull/6660/commits/f05a967dcfa13e4acc25a805871cb3f86880a284#diff-2feca7045c254673cb60781a052ce9d1R20)